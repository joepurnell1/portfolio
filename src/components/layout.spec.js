import React from 'react';
import renderer from 'react-test-renderer';
import Layout from './layout';

describe('Layout', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <Layout headerLink="https://www.youtube.com/watch?v=DUT5rEU6pqM">
        I never really knew that she could dance like this
      </Layout>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render inverted as expected', () => {
    const render = renderer.create(
      <Layout headerLink="https://www.youtube.com/watch?v=DUT5rEU6pqM" invert>
        She makes a man wants to speak Spanish
      </Layout>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
