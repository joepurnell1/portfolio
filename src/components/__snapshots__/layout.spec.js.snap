// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render default as expected 1`] = `
<div>
  <div
    className="sc-htpNat ezHVmo"
  >
    <a
      className="sc-bdVaJa ddjIOj"
      href="https://www.youtube.com/watch?v=DUT5rEU6pqM"
    >
      Say Hi
    </a>
  </div>
  I never really knew that she could dance like this
</div>
`;

exports[`Layout should render inverted as expected 1`] = `
<div>
  <div
    className="sc-htpNat hSsnMj"
  >
    <a
      className="sc-bdVaJa jUmgka"
      href="https://www.youtube.com/watch?v=DUT5rEU6pqM"
    >
      Say Hi
    </a>
  </div>
  She makes a man wants to speak Spanish
</div>
`;
